src/components/Vizmapper/VisualPropertyRender/Color.tsx

```diff
@@ -19,7 +19,171 @@ export function ColorPicker(props: {
 
   return (
     <Box>
-      <ChromePicker
+      <PhotoshopPicker
+        color={localColorValue}
+        onChange={(color: any) => {
+          setLocalColorValue(color.hex)
+          debouncedValueChange(color.hex)
+        }}
+      />
+    </Box>
+  )
+}
+
+export function ColorPickerCompact(props: {
+  currentValue: ColorType | null
+  onValueChange: (color: ColorType) => void
+}): React.ReactElement {
+  const { onValueChange, currentValue } = props
+  const debouncedValueChange = debounce(onValueChange, 200)
+
+  // use local state to appear instantaneous in the color picker,
+  // but the actual visual style model updates are debounced
+  const [localColorValue, setLocalColorValue] = React.useState<ColorType>(
+    currentValue ?? `#ffffff`,
+  )
+
+
+  const customColors = ['#ffffff', '#d3d2ff', '#d3d2ff', '#d3d2ff', '#ffcfd1', '#ffcfd1', '#ffcfd1', '#ffffcf', '#c5ffcf', '#c5ffcf', '#c5ffcf', '#c5ffcf',
+    '#d2d2d2', '#a4a2ff', '#a4a2ff', '#a4a2ff', '#ff9ba2', '#ff9ba2', '#ff9ba2', '#feff9b', '#7eff9c', '#7eff9c', '#7eff9c', '#7fffd0',
+    '#d2d2d2', '#63a5ff', '#746fff', '#ae6bff', '#ff5ea4', '#ff5f6e', '#ff9b6a', '#feff5e', '#7eff61', '#00ff62', '#00ff62', '#00ffd0',
+    '#a3a3a3', '#00a6ff', '#4035ff', '#b421ff', '#ff00a5', '#ff0030', '#ff9c1e', '#feff00', '#7dff00', '#00ff00', '#00ff63', '#00ffd1',
+    '#a3a3a3', '#00a7ff', '#1900ff', '#b600ff', '#ff00a5', '#ff0000', '#ff9c00', '#feff00', '#7dff00', '#00ff00', '#00ff63', '#00ffd1',
+    '#717171', '#00a7d5', '#1200d8', '#b500d7', '#e800a6', '#e70000', '#dba000', '#d1d300', '#94d500', '#00d800', '#00d86a', '#00d8d3',
+    '#717171', '#00a8a4', '#0b00a8', '#b400a7', '#b400a7', '#b40000', '#ac6e00', '#a2a400', '#a2a400', '#00a800', '#00a86f', '#00a8a4',
+    '#3a3a3a', '#007472', '#050075', '#7e0074', '#7d0000', '#7d0000', '#717200', '#717200', '#717200', '#1c7400', '#007472', '#007472',
+    '#000000', '#003c3b', '#01003c', '#41003c', '#41003c', '#3a3a00', '#808900', '#3a3a00', '#3a3a00', '#003c3b', '#003c3b', '#3a3a3a',
+  ]
+
+  return (
+    <Box>
+      <CompactPicker
+        colors={customColors}
+        color={localColorValue}
+        onChange={(color: any) => {
+          setLocalColorValue(color.hex)
+          debouncedValueChange(color.hex)
+        }}
+      />
+    </Box>
+  )
+}
+
+export function ColorPickerViridis(props: {
+  currentValue: ColorType | null
+  onValueChange: (color: ColorType) => void
+}): React.ReactElement {
+  const { onValueChange, currentValue } = props
+  const debouncedValueChange = debounce(onValueChange, 200)
+
+  // use local state to appear instantaneous in the color picker,
+  // but the actual visual style model updates are debounced
+  const [localColorValue, setLocalColorValue] = React.useState<ColorType>(
+    currentValue ?? `#ffffff`,
+  )
+
+  const customColors = [
+    ['#ffea00', '#98e21e', '#00ce71', '#00af91', '#008f9a', '#2f689b', '#55378d', '#560061'],
```

Comment:
These should be defined in the colors model impl so that they can be reused in other places in the app that needs to deal with colors.